Dockerfile

```diff
@@ -25,8 +28,9 @@ RUN apt-get update && \
     libxml2-dev \
     zlib1g-dev \
```

Comment:
Basically, we need all these libraries in the builder image too. We may not need the `-dev` version (which contains the headers), I'm not sure.

Reply:
Also these lines can be removed from here because I copy/pasted them from the official images, so they are already included in the parent image.